pkg/asset/machines/master.go

```diff
@@ -53,19 +53,40 @@ type Master struct {
 	UserDataFile       *asset.File
 	MachineConfigFiles []*asset.File
 	MachineFiles       []*asset.File
+
+	// SecretFiles is used by the baremetal platform to register the
+	// credential information for communicating with management
+	// controllers on hosts.
+	SecretFiles []*asset.File
+
+	// HostFiles is the list of baremetal hosts provided in the
+	// installer configuration.
+	HostFiles []*asset.File
 }
 
 const (
 	directory = "openshift"
 
-	// masterMachineFileName is the format string for constucting the master Machine filenames.
+	// secretFileName is the format string for constructing the Secret
+	// filenames for baremetal clusters.
+	secretFileName = "99_openshift-cluster-api_host-bmc-secrets-%s.yaml"
```

Comment:
I dunno why we like `99_` so much.  Maybe use `90_` for secrets to put them first and `95_` for hosts?

Reply:
Sure, I can do that.

Reply:
Since this merged, I submitted #2174 to address this comment.